content/docs/faq-structure.md

```diff
@@ -35,11 +35,11 @@ profile/
   ProfileAPI.js
 ```
 
-The definition of a "feature" is not universal, and it is up to you to choose the granularity. If you can't come up with a list of top-level folders, you can ask the users of your product what major parts it consists of, and use their mental model as a blueprint.
```

Comment:
You can change top-level folders to תיקיות ברמה עליונה. Tell me what do you think about it?